### DISCUSSION TITLE
Add primeflex to documentation as a dependancy for examples that use it

### DISCUSSION THREAD

**Discussion by Zynos-Logan** [primefaces/community#1295] [PrimeVue] [answered]
While starting to learn about primevue I found that when I copied the examples for different components such as the menu bar I wasn't seeing the same result in my project as it was displayed in the documentation. It was only after searching forum posts and combing the documentation that I saw primeflex listed. I believe noting by or within a comment in an example that uses primeflex that it is a dependency needed would improve the documentation greatly.

**Comment by tugcekucukoglu** [comment, marked as answer]
Menubar component does not use PrimeFlex but showcase demos might use PrimeFlex. also, could you share a forum post that you've found?
